tests/kuttl/tests/dataplane-deploy-no-nodes-test/01-assert.yaml

```diff
@@ -23,589 +23,3 @@ status:
-  namespace: openstack
-  ownerReferences:
-  - apiVersion: dataplane.openstack.org/v1beta1
-    blockOwnerDeletion: true
-    controller: true
-    kind: OpenStackDataPlaneRole
-    name: edpm-compute-no-nodes
-spec:
-  backoffLimit: 6
-  extraMounts:
-  - mounts:
-    - mountPath: /runner/env/ssh_key
-      name: ssh-key
-      subPath: ssh_key
-    - mountPath: /runner/inventory/hosts
-      name: inventory
-      subPath: inventory
-    - mountPath: /runner/network/nic-config-template
-      name: inventory
-      subPath: network
-    volumes:
-    - name: ssh-key
-      secret:
-        items:
-        - key: ssh-privatekey
-          path: ssh_key
-        secretName: dataplane-ansible-ssh-private-key-secret
-    - configMap:
-        items:
-        - key: inventory
-          path: inventory
-        - key: network
-          path: network
-        name: dataplanerole-edpm-compute-no-nodes
-      name: inventory
-  image: quay.io/openstack-k8s-operators/openstack-ansibleee-runner:latest
-  name: openstackansibleee
-  restartPolicy: Never
-  roles:
-    any_errors_fatal: true
-    become: true
-    gather_facts: false
-    hosts: all
-    name: Deploy EDPM Operating System Run
-    strategy: linear
-    tasks:
-    - import_role:
-        name: osp.edpm.edpm_sshd
-        tasks_from: run.yml
-      name: Run edpm_sshd
-      tags:
-      - edpm_sshd
-    - import_role:
-        name: osp.edpm.edpm_chrony
-        tasks_from: run.yml
-      name: Run edpm_chrony
-      tags:
-      - edpm_chrony
-    - import_role:
-        name: osp.edpm.edpm_chrony
-        tasks_from: online.yml
-      name: Run edpm_chrony (Online)
-      tags:
-      - edpm_chrony
-    - import_role:
-        name: osp.edpm.edpm_chrony
-        tasks_from: sync.yml
-      name: Run edpm_chrony (Sync)
-      tags:
-      - edpm_chrony
-    - import_role:
-        name: osp.edpm.edpm_timezone
-        tasks_from: run.yml
-      name: Run edpm_timezone
-      tags:
-      - edpm_timezone
-    - import_role:
-        name: osp.edpm.edpm_ovn
-        tasks_from: run.yml
-      name: Run edpm_ovn
-      tags:
-      - edpm_ovn
-    - name: Apply nftables configuration
-      import_role:
-        name: osp.edpm.edpm_nftables
-        tasks_from: run.yml
-      when: deploy_edpm_openstack_run_firewall|default(true)|bool
-      tags:
-        - edpm_firewall
-    - name: Run edpm_logrotate_crond
-      import_role:
-        name: osp.edpm.edpm_logrotate_crond
-        tasks_from: run.yml
-      tags:
-        - edpm_logrotate_crond
-    - name: Run edpm_iscsid
-      import_role:
-        name: osp.edpm.edpm_iscsid
-        tasks_from: run.yml
-      tags:
-        - edpm_iscsid
-  uid: 1001
-status:
-  JobStatus: Succeeded
-  conditions:
-  - message: AnsibleExecutionJob complete
-    reason: Ready
-    status: "True"
-    type: Ready
-  - message: AnsibleExecutionJob complete
-    reason: Ready
-    status: "True"
-    type: AnsibleExecutionJobReady
```

Comment:
why are you removing all of this? Can you explain in the commit message what caused the change and why this would fix it?